src/main/java/frc/auto/AutoBuilder.java

```diff
@@ -26,22 +26,22 @@ public static AutoBuilder getInstance() {
     }
 
     private AutoBuilder(){
-       writer = new CSVWriter(Constants.rootDirectory);
+       writer = new CSVWriter(Constants.ROOT_DIRECTORY);
        driveIO = DriveIO.getInstance();
-       driveIO.stopAllOutputs();
     }
 
     public void setStartRecording() {
         startTime = System.currentTimeMillis();
     }
     
     public void recordData() {
-
+        System.out.println("Made it to recordData()!");
         try {
             currentTime = System.currentTimeMillis();
             timer = currentTime - startTime;
 
             List<Double> rows = new ArrayList<Double>();
+            driveIO.updateInputs();
             rows.add((double)timer);
```

Comment:
I don't know if double will be large enough to hold the timer, usually, you would use a 'long' here. You may want to make a List of custom objects, here is an example class:

```
public class PathPoint {

    private final long timer;
    private final double[] values;

    public PathPoint(long timer, double[] values) {
        this.timer = timer;
        this.values = values;
    }

    public long getTimer() {
        return timer;
    }

    public double[] getValues() {
        return values;
    }
    
}
```

Reply:
A double and long are both 8 bytes in Java 
~ You should know this >:(